users/views.py

```diff
@@ -0,0 +1,104 @@
+from django.shortcuts import render
+
+import json
+import bcrypt
+import jwt
+
+import my_settings
+from django.views import View
+from django.http import JsonResponse
+
+from .models import Users, Gender
```

Comment:
```suggestion
from django.views import View
from django.http import JsonResponse

import my_settings
from .models import (
    Users, 
    Gender
)
```

한 줄에서 import 하고자 하는 대상이 두 개 이상일 경우 개별 줄로 분리해주는 것이 위코드 코딩 컨벤션입니다 :-)